src/NumberAdder.java

```diff
@@ -13,22 +13,21 @@ public static void main(String[] args) {
         List<Double> numbers = new ArrayList<>();
         Scanner input = new Scanner(System.in);
         double number = 0;
+        boolean exception = true;
```

Comment:
I don't think we need this exception variable. What is this supposed to help us with?

Reply:
i needed to define that variable for loop "while" without it loop doesn't work.

Reply:
it is needed for loop "while" without define loop doesn't work. And while is for exception which occur when user put for example leter against number.

Reply:
I still think we can write the code that will work the same without this exception var. Did you try to refactor it?